.github/workflows/build.yml

```diff
@@ -90,3 +87,24 @@ jobs:
         with:
           draft: true
           files: ${{ env.RELEASE_OUT }}/*
+
+  buildkit-edge:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver-opts: image=moby/buildkit:master
+          buildkitd-flags: --debug
+      -
+        name: Build
+        run: |
+          make release
```

Comment:
This doesn't conflict with the artifact creation etc?

Reply:
No it just runs the build without any release

Reply:
Ok, so it builds all the release artifacts but it doesn't upload them as there is no extra step in this workflow. Maybe add a comment for it as it is a bit confusing. Otherwise, we could just run some equivalent bake command directly as well.

Reply:
Done